Project-Aurora/Project-Aurora/Profiles/RocketLeague/Control_RocketLeague.xaml.cs

```diff
@@ -54,32 +49,19 @@ private void game_enabled_Checked(object sender, RoutedEventArgs e)
             }
         }
 
-        private void preview_team_SelectionChanged(object sender, SelectionChangedEventArgs e)
-        {
-            (profile_manager.Config.Event._game_state as GameState_RocketLeague).Player.Team = (PlayerTeam)this.preview_team.SelectedItem;
-        }
-
-        private void preview_boost_amount_ValueChanged(object sender, RoutedPropertyChangedEventArgs<double> e)
+        private void Button_Click(object sender, RoutedEventArgs e)
         {
-            if(sender is Slider)
-            {
-                this.preview_boost_amount_label.Text = (int)((sender as Slider).Value)+"%";
-
-                if(IsLoaded)
-                    (profile_manager.Config.Event._game_state as GameState_RocketLeague).Player.BoostAmount = (int)((sender as Slider).Value);
-            }
+            Process.Start(@"https://bakkesmod.com/index.php");
         }
 
-        private void preview_team1_score_ValueChanged(object sender, RoutedPropertyChangedEventArgs<object> e)
+        private void Button_Click_1(object sender, RoutedEventArgs e)
         {
-            if(IsLoaded && sender is IntegerUpDown && (sender as IntegerUpDown).Value.HasValue)
-                (profile_manager.Config.Event._game_state as GameState_RocketLeague).Match.OrangeTeam_Score = (sender as IntegerUpDown).Value.Value;
+            Process.Start(@"https://github.com/diogotr7/AuroraGSI-RocketLeague/releases");
         }
 
-        private void preview_team2_score_ValueChanged(object sender, RoutedPropertyChangedEventArgs<object> e)
+        private void Button_Click_2(object sender, RoutedEventArgs e)
```

Comment:
I'd change the names of these functions to something which details more about what they're for